google/resource_bigquery_table.go

```diff
@@ -394,3 +428,24 @@ func flattenTimePartitioning(tp *bigquery.TimePartitioning) []map[string]interfa
 
 	return []map[string]interface{}{result}
 }
+
+func expandView(configured interface{}) *bigquery.ViewDefinition {
+	raw := configured.([]interface{})[0].(map[string]interface{})
+	vd := &bigquery.ViewDefinition{Query: raw["query"].(string)}
+
+	if v, ok := raw["use_legacy_sql"]; ok {
+		vd.UseLegacySql = v.(bool)
+	}
+
+	return vd
+}
+
+func flattenView(vd *bigquery.ViewDefinition) []map[string]interface{} {
+	result := map[string]interface{}{"query": vd.Query}
+
+	if vd.UseLegacySql {
```

Comment:
Fun fact: this already actually gets stored in state whether it's true or false because of the way terraform handles objects, so we may as well just simplify the logic here and always set it anyway.